hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1583,11 +1583,11 @@ private void checkAcls(ResourceType resType, StoreType store,
    *
    * @return true if permission granted, false if permission denied.
    */
-  private boolean hasAcls(ResourceType resType, StoreType store,
-      ACLType acl, String vol, String bucket, String key) {
+  private boolean hasAcls(String userName, ResourceType resType,
+      StoreType store, ACLType acl, String vol, String bucket, String key) {
     try {
       return checkAcls(resType, store, acl, vol, bucket, key,
-          ProtobufRpcEngine.Server.getRemoteUser(),
+          UserGroupInformation.createRemoteUser(userName),
```

Comment:
Question for @xiaoyuyao here: `UserGroupInformation.createRemoteUser()` create a new UGI with `AuthMethod.SIMPLE`. Does this matter?

IMO in this case only username matters.

Reply:
That should be fine in this case given the user has passed the RPC layer Kerberos auth.

Reply:
Sounds good. Thanks!